product_docs/docs/pgd/5/twophase.mdx

```diff
@@ -19,14 +19,14 @@ software components:
 
 -   An application program (AP) that defines transaction boundaries and specifies
     actions that constitute a transaction
--   Resource managers (RMs, such as databases or file-access systems) that provide
+-   Resource managers (RMs), such as databases or file-access systems, that provide
     access to shared resources
```

Comment:
Query as to why we are showing these abbreviations when they aren't used anywhere else in this topic.